src/platformClient.js

```diff
@@ -63,8 +63,12 @@ class PlatformClient {
 
   async createHeadersWithResolvedToken(headers = {}) {
     if (this.tokenResolverFunction) {
-      let token = await this.tokenResolverFunction();
-      headers.Authorization = `Bearer ${token}`;
+      if (headers.Authorization) {
```

Comment:
`Authorization` is actually case insensitive, this likely isn't good enough, but perhaps it is for now.